cross/nzbhydra2/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = NZBHydra2
+PKG_VERS = 1.4.8
+PKG_REPO = theotherp
+PKG_PLAT = linux
```

Comment:
Not needed, just use it directly.

Reply:
removed PKG_PLAT and using "linux" directly now